crates/abi/abi/HEVM.sol

```diff
@@ -84,6 +86,9 @@ record()
 accesses(address)(bytes32[], bytes32[])
 skip(bool)
 
+recordStateDiff()
+getStateDiff()(AccountAccess[])
```

Comment:
We should rename this to `getRecordedStateDiff` to match `getLogs` and `getRecordedLogs`